tachys/src/html/element/elements.rs

```diff
@@ -0,0 +1,488 @@
+    textarea HtmlTextAreaElement [autocomplete, cols, dirname, disabled, form, maxlength, minlength, name, placeholder, readonly, required, rows, wrap] true,
+    /// The `<tfoot>` HTML element defines a set of rows summarizing the columns of the table.
+    tfoot HtmlTableSectionElement [] true,
+    /// The `<th>` HTML element defines a cell as header of a group of table cells. The exact nature of this group is defined by the scope and headers attributes.
+    th HtmlTableCellElement [abbr, colspan, headers, rowspan, scope] true,
+    /// The `<thead>` HTML element defines a set of rows defining the head of the columns of the table.
+    thead HtmlTableSectionElement [] true,
+    /// The `<time>` HTML element represents a specific period in time. It may include the datetime attribute to translate dates into machine-readable format, allowing for better search engine results or custom features such as reminders.
+    time HtmlTimeElement [datetime] true,
+    ///	The `<title>` HTML element defines the document's title that is shown in a Browser's title bar or a page's tab. It only contains text; tags within the element are ignored.
+    title HtmlTitleElement [] true,
+    /// The `<tr>` HTML element defines a row of cells in a table. The row's cells can then be established using a mix of td (data cell) and th (header cell) elements.
+    tr HtmlTableRowElement [] true,
+    /// The `<u>` HTML element represents a span of inline text which should be rendered in a way that indicates that it has a non-textual annotation. This is rendered by default as a simple solid underline, but may be altered using CSS.
+    u HtmlElement [] true,
+    /// The `<ul>` HTML element represents an unordered list of items, typically rendered as a bulleted list.
+    ul HtmlUListElement [] true,
+    /// The `<var>` HTML element represents the name of a variable in a mathematical expression or a programming context. It's typically presented using an italicized version of the current typeface, although that behavior is browser-dependent.
+    var HtmlElement [] true,
+    /// The `<video>` HTML element embeds a media player which supports video playback into the document. You can use `<video>` for audio content as well, but the audio element may provide a more appropriate user experience.
+    video HtmlVideoElement [controls, controlslist, crossorigin, disablepictureinpicture, disableremoteplayback, height, r#loop, muted, playsinline, poster, preload, src, width] true,
+}
+
+pub fn option<Rndr>() -> HtmlElement<Option_, (), (), Rndr>
+where
+    Rndr: Renderer,
+{
+    HtmlElement {
+        tag: Option_,
+        rndr: PhantomData,
+        attributes: (),
+        children: (),
+        #[cfg(debug_assertions)]
+        defined_at: std::panic::Location::caller(),
+    }
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct Option_;
+
+impl ElementType for Option_ {
+    type Output = web_sys::HtmlOutputElement;
```

Comment:
Also, `Option_` incorrectly uses `HtmlOutputElement` instead of `HtmlOptionElement`.